interfaces/cython/cantera/examples/onedim/diffusion_flame_batch.py

```diff
@@ -122,16 +137,24 @@ def interrupt_extinction(t):
     try:
         # Try solving the flame
         f.solve(loglevel=0)
-        file_name = 'pressure_loop_' + format(p, '05.1f') + '.xml'
-        f.save(data_directory + file_name, name='solution', loglevel=1,
-               description='Cantera version ' + ct.__version__ +
-               ', reaction mechanism ' + reaction_mechanism)
+        if hdf_output:
+            group = 'pressure_loop_{:05.1f}'.format(p)
```

Comment:
Maybe a nice place to use nested HDF5 groups?
```suggestion
            group = 'pressure_loop/{:05.1f}'.format(p)
```
Likewise with `strain_loop`.